js/pages/prediction/utils.js

```diff
@@ -25,6 +25,15 @@ define(
         }
     }
 
+    function getDefaultModelSettingName(defaultModelSettings, settingName) {
+        const settingList = defaultModelSettings.filter(item => item.setting === settingName);
```

Comment:
Why not
```
const setting = defaultModelSettings.find(item => item.setting === settingName);
return setting ? settingList[0].name : "SETTING NAME NOT FOUND!!";
```
?

Shouldn't it throw exception if the setting is not found?

Reply:
Agreed that it should throw an exception - I'll make this change as well.